execution_engine/src/shared/newtypes/macros.rs

```diff
@@ -99,6 +99,7 @@ macro_rules! make_array_newtype {
             }
         }
 
+        #[allow(unused_qualifications)]
```

Comment:
Why did this come up?

Reply:
The ee_699 test was the trigger.  It imports `bytesrepr` which make the macro expansion trigger the `unused_qualifications` warning.  I think it's probably OK to disable that warning inside a macro.

Although, now you've made me look closer - seems like that macro is only used inside that one test file.  We should move it into the test module at some point.